src/main/java/org/mdbenefits/app/preparers/HouseholdDetailsPreparer.java

```diff
@@ -19,68 +19,27 @@
         var household = (List<Map<String, Object>>) submission.getInputData().get("household");
         if (household != null) {
             for (int i = 0; i < household.size(); i++) {
                 Map<String, Object> householdMember = household.get(i);
-                var educationStatus = householdMember.get("householdMemberHighestEducation");
-                results.put("householdHighestEducation" + i, new SingleField("householdHighestEducationFormatted",
-                        SubmissionUtilities.PDF_EDUCATION_MAP.get(educationStatus), i + 1));
-
-                var maritalStatus = householdMember.get("householdMemberMaritalStatus");
-                results.put("householdMaritalStatus" + i, new SingleField("householdMaritalStatusFormatted",
-                        SubmissionUtilities.PDF_MARITAL_STATUS_MAP.get(maritalStatus), i + 1));
+                String fullName = String.format("%s, %s", householdMember.get("householdMemberLastName"), householdMember.get("householdMemberFirstName"));
+                results.put("householdMemberFullName_" + i, new SingleField("householdMemberFullName",
+                        fullName, i + 1));
 
-                var relationshipStatus = householdMember.get("householdMemberRelationship");
-                results.put("householdRelationship" + i, new SingleField("householdRelationshipFormatted",
-                        SubmissionUtilities.PDF_RELATIONSHIP_MAP.get(relationshipStatus), i + 1));
-
-                var birthday = Stream.of("householdMemberBirthMonth", "householdMemberBirthDay", "householdMemberBirthYear")
+                var dob = Stream.of("householdMemberBirthMonth", "householdMemberBirthDay", "householdMemberBirthYear")
                         .map(householdMember::get)
                         .reduce((e, c) -> e + "/" + c)
                         .get();
-                results.put("householdBirthday" + i, new SingleField("householdBirthdayFormatted", (String) birthday, i + 1));
-
-                String uuid = (String) householdMember.get("uuid");
-                results.put("householdUSCitizen" + i, new SingleField("householdUSCitizenDerived",
-                        nonCitizens != null && nonCitizens.contains(uuid) ? "No" : "Yes", i + 1));
+                results.put("householdMemberDOB_" + i, new SingleField("householdMemberDOB", (String) dob, i + 1));
 
-                if (submission.getInputData().getOrDefault("permissionToAskAboutRace", "false").equals("true")) {
-                    results.put("householdRaceEthnicityCode" + i,
-                            new SingleField("householdRaceEthnicityCode", getRaceEthnicityCode(submission, uuid), i + 1));
-                }
+                results.put("householdMemberSSN_" + i, new SingleField("householdMemberSSN", SubmissionUtilities.formatSSN((String) householdMember.get("householdMemberEncryptedSSN")), i + 1));
```

Comment:
same here for the "householdMemberEncryptedSSN" - you'll want the unencrypted version.  Unless we are working with the encrypted version of the SSN? That doesn't make sense though.